src/structures/User.js

```diff
@@ -222,11 +222,15 @@ class User extends Base {
 
   /**
    * Creates a DM channel between the client and the user.
+   * @param {boolean} [forceFetch=false] Whether to skip the cache check and request the API
    * @returns {Promise<DMChannel>}
    */
-  async createDM() {
-    const { dmChannel } = this;
-    if (dmChannel && !dmChannel.partial) return dmChannel;
+  async createDM(forceFetch = false) {
```

Comment:
This might be better named as forceCreate?

Reply:
impl with https://github.com/discordjs/discord.js/pull/4592/commits/5daa176142c067712cb9aeaaa52733083a8adb56
cc: @kyranet